gammapy/scripts/config/schema.yaml

```diff
@@ -186,6 +186,57 @@ properties:
             dataset-type:
                 enum: [not assigned, SpectrumDatasetOnOff, MapDataset]
             containment_correction: {type: boolean}
+            offset-max: {type: string}
```

Comment:
This is actually a number, because astrropy quantities are considered as numbers in the validation as in the case for `lon` and `lat` for the observations filtering above.

Reply:
Done